package Aufgabe_4;

public class eigeneException extends Exception{

    public eigeneException(){

        super("Das Format ihrer Mailadresse ist ung√ºltig");
    }
}

// Wird gleich bei Aufgabe 3 als Exception verwendet somit ist die Aufgabe 3 der Test
